.yamato/config/__shared.metafile

```diff
@@ -115,15 +133,7 @@ platforms:
     os: android
     apis:
       - name: OpenGLES3
-        exclude_test_platforms:
```

Comment:
should these be added back, or we want editmodes/playmodes on mobiles?

Reply:
see prev comments